filtered-resource-service/src/main/java/uk/gov/gchq/palisade/service/filteredresource/message/AuditMessage.java

```diff
@@ -128,7 +129,7 @@ public String getServerIP() {
     }
 
     @Generated
-    public String getServerHostName() {
+    public String getServerHostname() {
```

Comment:
Does this really need to be  changed?

Reply:
Yep, the field is serverHostname, so the getter must be getServerHostname, otherwise spring fails to (de)serialise the messages

Reply:
Ah, that makes sense.